docs/accounts/tenant/people.md

```diff
@@ -26,4 +26,11 @@ A user can be invited as either a **Member** or an **Owner**.
 
 To invite a new tenant member, navigate to **People** from the **Tenant Settings** section. Click **Invite Member**. Enter the email address of the user you wish to invite and select the role you wish to assign them. Click **Invite**.
 
-<img src="/images/docs/pipes/pipes-enterprise-people-invite-member.png" width="400pt"/>
\ No newline at end of file
+<img src="/images/docs/pipes/pipes-enterprise-people-invite-member.png" width="400pt"/>
+
+## Last Activity
+
+The **Last Activity** column represents the most recent date on which an activity was performed by a user in the tenant. The column serves as a timestamp to track the latest user engagement within the tenant, providing insights into user activity and system usage trends. The following actions currently count as user activity for a tenant:
```

Comment:
To be honest, I think your previous one was better - It was vague, but I dont like the bullets here, and its incomplete anyway. Let's use the previous one for now. I feel like we need to rework this whole page at some point anyway.